### DISCUSSION TITLE
How can I find the original AUPRO implementation

### DISCUSSION THREAD

**Discussion by ByChelsea** [open-edge-platform/anomalib#895] [Q&A] [answered]
@samet-akcay @djdameln  @ashwinvaidya17 
This code is not complete. I have opened this thread to discuss AUPRO implementation details. I have two opens:
- Do we need AUROC and F1 for segmentation as it does not add much value? Can we replace it with just AUPRO metric? 
- The original repo (from which this code was borrowed) contains two skimage functions - label, regionprops. Should we implement torch version of these methods to make this metric torch compatible?

_Originally posted by @blakshma in https://github.com/openvinotoolkit/anomalib/issues/84#issuecomment-1020763261_
            
Hi, I’m sorry to bother you, could you please tell me the link to the original code repo of AUPRO?

**Comment by mjack3** [comment, marked as answer]
AUPRO can be achieved from MVTec AD dataset
